src/util/Util.js

```diff
@@ -13,6 +13,43 @@ class Util {
   }
 
   /**
+   * Flatten an object. Any properties that are collections will get converted to an array of keys.
+   * @param {Object} obj The object to flatten.
+   * @param {Array<Object<string, boolean|string>>} [props=[]] Specific properties to include/exclude.
```

Comment:
After discussion about this param in staff chat, it should be: `...Object<string, boolean|string>`